.github/workflows/publish.yml

```
@@ -1,32 +1,23 @@
-name: CI
-
-on: [push]
-
+name: publish
+on: push
 jobs:
-  default:
+  main:
     name: Publish CI tagged release for MDX
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0 # for lerna
-
-      - uses: actions/setup-node@v2
-        with:
-          node-version: 14
-
-      - name: Install Dependencies
-        run: yarn --frozen-lockfile
+      - uses: dcodeIO/setup-node-nvm@master
```

please pin a specific major version range

```suggestion
      - uses: dcodeIO/setup-node-nvm@v4
```